src/ansible_builder/containerfile.py

```diff
@@ -144,7 +145,7 @@ def prepare(self) -> None:
         self._insert_global_args()
 
         if image == "base":
-            self.steps.append("RUN $PYCMD -m pip install --no-cache-dir bindep pyyaml requirements-parser")
+            self.steps.append("RUN $PYCMD -m pip install --no-cache-dir bindep pyyaml")
```

Comment:
We need `packaging` in the container too correct?  Since it's working now, I assume it's being installed as a dep somewhere, but we probably want to explicitly include it here incase something changes with deps.

Reply:
Hrm, yeah, I think you might be right about that since introspect needs it.

Reply:
Done.